homeassistant/components/ffmpeg.py

```diff
@@ -203,11 +161,23 @@ def async_run_test(self, input_source):
 class FFmpegBase(Entity):
     """Interface object for ffmpeg."""
 
-    def __init__(self, initial_state=True):
+    def __init__(self, hass, initial_state=True):
         """Initialize ffmpeg base object."""
+        self.hass = hass
         self.ffmpeg = None
         self.initial_state = initial_state
 
+        # init dispatcher for service routing
+        async_dispatcher_connect(
```

Comment:
Constructors should be able to be used in both sync and async worlds. Prefer if you move this to either `async_added_to_hass` or call a method after instantiating it.